easybuild/easyconfigs/u/unidata/unidata-2.8-package-data.patch

```diff
@@ -0,0 +1,99 @@
+diff -ru unidata-2.8/Lib/__init__.py unidata-2.8-edit/Lib/__init__.py
```

Comment:
Please add patch description + link to upstream source or mention patch author (+ fix SHA256 for this patch file)